release-notes/major9/09.17.0/valtimo-backend-libraries.md

```diff
@@ -23,6 +23,9 @@ The following features were added:
 
   Properties in plugins can now be marked as secret. When configuring the plugin the secrets will be encrypted before 
   being stored.
+
+  This functionality requires an application property `valtimo.plugin.encryption-secret` the value of this property will
```

Comment:
Maybe mention this under known issues as well as the number of characters (16) required for this property.